subsys/esb/esb.c

```diff
@@ -243,6 +243,48 @@ static uint32_t addr_conv(const uint8_t *addr)
 	return __REV(bytewise_bit_swap(addr));
 }
 
+static void apply_errata143_workaround(void)
+{
+#ifdef CONFIG_SOC_NRF52832
```

Comment:
If you would include `nrf_erratas.h` then you can leverage `NRF52_ERRATA_143_PRESENT` macro and function for runtime errata need test `nrf52_errata_143(void)`

Reply:
Ok